examples/using-gatsby-image/src/pages/traced-svg.js

```diff
@@ -57,8 +57,9 @@ const UnsplashMasonry = edges => (
             "& .gatsby-image-wrapper:hover": {
               "& div + img": {
                 opacity: `1 !important`,
+                transition: `none !important`,
               },
-              "& img + img": {
+              "& img + picture > img": {
```

Comment:
Above brings back this (right image is hovered): 
![image](https://user-images.githubusercontent.com/21834/47399072-39575d00-d737-11e8-94a9-3cc14b70468d.png)